modules/network-observability-total-resource-usage.adoc

```diff
@@ -5,11 +5,10 @@
 [id="network-observability-total-resource-usage-table_{context}"]
 = Total average memory and CPU usage
 
-The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400 for 3 different tests: `Test 1`, `Test 2`, and `Test 3`. The tests differ in the following ways:
+The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400 for 2 different tests: `Test 1`, `Test 2`. The tests differ in the following ways:
 
-- `Test 1` takes into account the total number of namespace, pods and services in an {product-title} cluster, places load on the eBPF agent, and represents use cases with a high number of workloads for a given cluster size. For example, `Test 1` consists of 76 Namespaces, 5153 Pods, and 2305 Services.
-- `Test 2` takes into account a high ingress traffic volume.
-- `Test 3` takes into account the total number of namespace, pods and services in an {product-title} cluster, places load on the eBPF agent on a larger scale than `Test 1`, and represents use cases with a high number of workloads for a given cluster size. For example, `Test 3` consists of 553 Namespaces, 6998 Pods, and 2508 Services.
+- `Test 1` takes into account high ingress traffic volume in addition to the total number of namespace, pods and services in an {product-title} cluster, places load on the eBPF agent, and represents use cases with a high number of workloads for a given cluster size. For example, `Test 1` consists of 76 Namespaces, 5153 Pods, and 2305 Services.
+- `Test 2` takes into account high ingress traffic volume in addition to the total number of namespace, pods and services in an {product-title} cluster, places load on the eBPF agent on a larger scale than `Test 1`, and represents use cases with a high number of workloads for a given cluster size. For example, `Test 2` consists of 553 Namespaces, 6998 Pods, and 2508 Services.
 
 Since different types of cluster use cases are exemplified in the different tests, the numbers in this table cannot be linearly compared side-by-side, but instead are intended to be used as a benchmark for evaluating your personal cluster usage. The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
```

Comment:
I think the phrase "Since different types of cluster use cases are exemplified [...] for evaluating your personal cluster usage." was previously added to explain why numbers did not seem logical if the reader expected some linearity (e.g. the test 2 / 65 nodes showed lower CPU than test 1 / 25 nodes)
It's not true anymore, with Test 2 being merged in Test 1 and Test3. So I think we can remove it - do you agree?
cc @stleerh @memodi

Reply:
On a different note (since you decided to keep the paragraph), the wording for Test 2 could be shorter and avoids the user having to compare the words to Test 1.  Suggest:

- `Test 2` is the same as `Test 1` except on a larger scale. For example, `Test 2` consists of 553 Namespaces, 6998 Pods, and 2508 Services.

Reply:
I was just waiting to see if you and Mehul agree to remove since Joel asked if you guys agree. If everyone thinks it should be removed, or most, then I can remove it 😅 But your suggestion is good too and definitely shortens the introductory text.

Reply:
It may as well mentioned the nodes too just to be explicit and complete.

`Test 1` consists of 25 Nodes, 76 Namespaces, 5153 Pods, and 2305 Services.
`Test 2` consists of 250 Nodes, 553 Namespaces, 6998 Pods, and 2508 Services.

Reply:
I agree with @jotak's and @stleerh's suggestions here, we could also add scale of Network traffic that we measure for each test, Test 1 has ~350 MB/s and Test 2 has ~950 MB/s